README.md

```diff
@@ -2,33 +2,16 @@
 
 [**Add the official Cordlr to your server**](https://discordapp.com/oauth2/authorize?client_id=223165778357256194&scope=bot&permissions=0x00000008)
 
+## Requirements
+
+Devcord requires ES2015. So the latest, suggested versions are all versions up from `6.x`
+
 ## Installation
-* `npm install`
-* `node index.js`
-
-## Roadmap
-
-### Commands
-* [x] Selfassignable Roles
-* [ ] Caniuse Commands
-* [ ] Devdocs Documentation Commands
-* [ ] Banning and Kicking
-
-### Integrations
-* [ ] Github
-  * [ ] Commit History (via Hooks)
-  * [ ] Pull Request Notification (via Hooks)
-  * [ ] Comment Notifications (via Hooks)
-* [ ] Gitlab
-  * [ ] see **Github**
-* [ ] Bitbucket
-  * [ ] see **Github**
-
-### Features
-* [ ] Backend Webinterface with Discord Login
-  * [ ] shows server stats
-  * [ ] ban history
-  * [ ] kick history
+* Install node packages via `npm install`

```

Comment:
This bot can be installed off GitHub via:

```
npm install -g devcord/cordlr
```

Other packages can be installed their own ways.  Most likely through npm as well.